### ISSUE TITLE
Show actual annotations in thumbnails

### ISSUE THREAD

**Issue by leowe** [scalableminds/webknossos#6332]
## Detailed Description
In #3937, we decided to only display the datasource thumbnails for the annotations instead of the actual annotations due to several reasons (which position to show in the thumbnail, how to get all layers from the data set as well, apply mappings, how to fetch the data so that in the buckets where there is no annotation the segmentation is loaded).
However, the annotation thumbnails should  show the true annotations as well.

**Comment by normanrz** [comment]
> which position to show in the thumbnail

that should probably be the annotation position. i think, we also store the zoom

**Comment by fm3** [comment]
Yes, annotation position, rotation, zoom should be used, the hard part here is to get this information from the tracingstore to the thumbnail renderer :D This has to go around quite a few corners. But I still think it might be worth it.

**Comment by hotzenklotz** [comment]
@fm3 With the many improvements around the thumbnail renderer lately (social media preview, etc), is this still an open issue? Or can we close it?

**Comment by fm3** [comment]
I’d still say this would be nice to have. Not sure how the priority is. Currently, for annotations in publications, their dataset’s thumbnail is shown in place of the actual annotation.
